cablelabs/lpwanserver
# - Fixed #156 Integrations now getting set in LV2 and for LV2 Applicat…

…ions

- Fixed #148, where keys were not getting set in LV2.  The appKey has to be changed to the nwkKey when going from 1.0.x to 1.1 servers.

#### What does this PR do?
Fixes bug where integrations did now get set up in LoraOSV2 when pulling applications
Fixes bug where NwkKey wasn't being set in LoraOSV2 when pushing applications from other networks. 
#### Do you have any concerns with this PR?
No
#### How can the reviewer verify this PR?
Run the demo and verify the LoraV2 server has integrations set to `<lpwanserver>/api/ingest/<appId>/<networkId>`
and LPWan is set to 
`http://localhost:9999`

Also the Keys for the TTN devices should be visible in the LoraV2 Server

#### Any background context you want to provide?
No
#### Screenshots or logs (if appropriate)
![image](https://user-images.githubusercontent.com/4982278/46262247-41b7d180-c4c4-11e8-99cb-a92673dff143.png)

![image](https://user-images.githubusercontent.com/4982278/46262268-6e6be900-c4c4-11e8-9462-8fa5785b11ca.png)

![image](https://user-images.githubusercontent.com/4982278/46262279-a2dfa500-c4c4-11e8-8060-b0fe5584b34a.png)

![image](https://user-images.githubusercontent.com/4982278/46262282-b428b180-c4c4-11e8-9982-89cf091cb76d.png)

#### Questions:
- Have you connected this PR to the issue it resolves?
Yes #148 #156 
- Does the documentation need an update?
No
- Does this add new dependencies?
No
- Have you added unit or functional tests for this PR?
Yes
